packages/webapp/src/components/Form/Unit/index.jsx

```diff
@@ -188,7 +193,7 @@ const Unit = ({
     const options = getOptions(unitType, system);
     const hookFormValue = hookFormGetValue(name);
     const value = hookFormValue || (hookFormValue === 0 ? 0 : defaultValue);
-    const isSelectDisabled = options.length <= 1;
+    const isSelectDisabled = options.length <= 1 || selectDisabled;
```

Comment:
The previous expression should be sufficient. Once the irrigation_depth has only one option in util/convert-units this should work as expected.